pkgs/applications/science/logic/eprover/default.nix

```diff
@@ -17,6 +17,7 @@ stdenv.mkDerivation rec {
   configureFlags = [
     "--exec-prefix=$(out)"
     "--man-prefix=$(out)/share/man"
+    "--enable-ho"
```

Comment:
This is unfortunately a breaking change, since it no longer builds the `eprover` binary.

I'm not sure what the best solution here is.  We could add an `enableHO` flag to the derivation, add a `eprover-ho = eprover.override { enableHO = true; };` attribute and have isabelle depend on `eprover-ho`.

Or we could `ln -s eprover-ho eprover` and hope that the first-order support still works the same as before (small performance differences aside).

Reply:
I could also just remove that part and continue using the version that comes with Isabelle, there is no _need_ to use the nixpkgs version like with veriT

Reply:
We definitely want to package ehoh anyhow, but `eprover` should still be available.  This can also be in a different PR though.

Reply:
I've added an option to the derivation now